packages/server-core/src/projects/project-check-unfetched-commit/project-check-unfetched-commit.test.ts

```diff
@@ -71,12 +74,15 @@ describe('project-check-unfetched-commit.test', () => {
 
     testUserApiKey = await app.service(userApiKeyPath).create({ userId: testUser.id })
 
-    await app.service(identityProviderPath).create(
+    await app.service(identityProviderPath)._create(
```

Comment:
You can use `identityProviderDataResolver.resolve()` method here to populate default values.